src/test/RUNTESTS

```diff
@@ -233,11 +234,17 @@ done
 echo RUNTESTS: test-type: $testtype
 
 # check if timeout supports "-k" option
-timeout -k 0s echo 2>/dev/null
-if [ $? ]; then
+timeout -k 1s 1s true
```

Comment:
I would leave `2>/dev/null` to avoid printing error messages on the console (in case your version of `timeout` does not support `--foreground` or `-k` option).